core-web/libs/data-access/src/lib/dot-contentlet/dot-contentlet.service.ts

```diff
@@ -23,4 +23,18 @@ export class DotContentletService {
             })
             .pipe(take(1), pluck('entity', 'versions', language));
     }
+
+    /**
+     * Get Contentlet permissions data
+     * @param string identifier
+     * @returns Observable<{ [key: string]: string[] }
+     * @memberof DotContentletService
+     */
+    getContentletPermissions(identifier: string): Observable<{ [key: string]: string[] }> {
```

Comment:
Don't we defined keys for the permissions?

Reply:
fixed!